### PR TITLE
Tiny changes to tutorial

### PR DESCRIPTION
**Pull request by dummycode**
**Changes**
- Was reading through the basic tutorial and saw two tiny things that bugged me, so I figured I would submit them

### CODE REVIEW

File: docs/tutorial/tutorial.md [hunk 1]
```diff
@@ -877,7 +877,7 @@ We also need to change it a little, since Game state is structured differently.
     squares[i] = this.state.xIsNext ? 'X' : 'O';
     this.setState({
       history: history.concat([{
-        squares: squares
+        squares,
```
Comment (dummycode) [inline docs/tutorial/tutorial.md]: Object shorthand. Can revert if wanting to keep things simple.

File: docs/tutorial/tutorial.md [hunk 2]
```diff
@@ -1029,7 +1029,7 @@ Add a method called `jumpTo` to the Game class:
   jumpTo(step) {
     this.setState({
       stepNumber: step,
-      xIsNext: (step % 2) ? false : true,
+      xIsNext: !(step % 2),
```
Comment (dummycode) [inline docs/tutorial/tutorial.md]: Why even bother with the ternary operator?

Comment (dummycode) [inline docs/tutorial/tutorial.md]: Will need to update linked codepen after this change.

**Comment by dummycode** [comment]
Worth a look? @gaearon

File: docs/tutorial/tutorial.md (hunk 2 above)
Comment (nhunzaker) [inline docs/tutorial/tutorial.md]: I find having a lot of operators in a tight space can be hard to parse. I like it as a ternary.

File: docs/tutorial/tutorial.md (hunk 1 above)
Comment (nhunzaker) [inline docs/tutorial/tutorial.md]: I could go either way, but I'd like to lean on the side of simple for a tutorial.

File: docs/tutorial/tutorial.md (hunk 2 above)
Comment (gaearon) [inline docs/tutorial/tutorial.md]: I would rewrite this as `xIsNext: (step % 2) === 0`. That's the intended logic right?

File: docs/tutorial/tutorial.md (hunk 1 above)
Comment (gaearon) [inline docs/tutorial/tutorial.md]: Let's keep it ES5-ish.

File: docs/tutorial/tutorial.md (hunk 2 above)
Comment (nhunzaker) [inline docs/tutorial/tutorial.md]: :+1:

Comment (dummycode) [inline docs/tutorial/tutorial.md]: Cool, changes made. Sorry this ended up being such a nit picky PR. Thanks for the feedback!

**Comment by gaearon** [comment]
Thanks!
